DependencyInjection/Configuration.php

```diff
@@ -65,41 +64,27 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('document_tree')
-                    ->useAttributeAsKey('class')
-                    ->prototype('array')
-                        ->fixXmlConfig('valid_child', 'valid_children')
-                        ->children()
-                            ->arrayNode('valid_children')
-                                ->prototype('scalar')->end()
-                                ->info('class names of valid children, manage tree operations for them and hide other children')
-                            ->end()
-                            ->scalarNode('image')
-                                ->defaultValue('')
-                            ->end()
-                        ->end()
-                    ->end()
-                 ->end()
-
-                 ->scalarNode('document_tree_repository')->defaultValue('default')->end()
-
-                ->arrayNode('document_tree_defaults')
-                    ->prototype('scalar')->end()
-                ->end()
-
-                ->arrayNode('document_tree_options')
-                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
                     ->children()
-                        ->integerNode('depth')
-                            ->defaultValue(1)
-                            ->info('Depth to which to fetch tree children when rendering the initial tree')
+                        ->scalarNode('repository_name')
+                            ->defaultValue('default')
+                            ->info('The repository name the resource API connects to.')
+                        ->end()
+                        ->arrayNode('routing_defaults')
+                            ->prototype('scalar')->end()
                         ->end()
-                        ->scalarNode('precise_children')
-                            ->defaultTrue()
-                            ->info('Exact check if document has children. For large trees, set to false for better performance, but user might needs to click on expand to see there are no children.')
+                        ->scalarNode('sortable_by')
+                            ->defaultValue('position')
+                            ->info('Defines the document property siblings are sorted.')
                         ->end()
-                        ->booleanNode('confirm_move')
-                            ->defaultTrue()
-                            ->info('Whether moving a node in the tree asks for confirmation.')
+                        ->arrayNode('move')
+                            ->canBeEnabled()
+                            ->children()
+                                ->booleanNode('reorder')
+                                    ->defaultTrue()
+                                    ->info('Ordering after a move action is enabled, causes new requests.')
```

Comment:
as this is the config for the tree on the dashboard, i think i would just drop that configuration here and set the value to true in the DI extension. if somebody does a custom tree somewhere else its ok if controller and template support disabling the reorder but for the dashboard i don't see a use case where it would make sense to move without reorder. then that whole discussion can remain in the resource bundle doc and the admin doc does not need to bother with the details of tree things.

and i guess some upcoming resource bundle version should support a combined move and reorder operation to go to 1 instead of 3 requests, but that is post 2.0 release...